### PR TITLE
Made simulator put events under correct node

### PR DESCRIPTION
**Pull request by local-minimum**
Before the events didn't end up under the `events` node.

Fixed it.

Tested by running simulator and making sure `/events` listed the event.

Seems fine.

### CODE REVIEW

**Comment by sgstair** [comment]
Looks like it should work without issues. Thanks!

File: sandbox/simulator/main [hunk 1]
```diff
@@ -100,6 +100,17 @@ define("TOP_STATS",10); // keep top 10 stats
 
 // Figure out event index to use for this run, and start loggers.
 
+$events_root = db_QueryFetch(
+		"SELECT
+			n.id,
+			".DB_FIELD_DATE('n.modified', 'modified')."
```
Comment (mikekasprzak) [inline sandbox/simulator/main]: Date is unused.

File: sandbox/simulator/main [hunk 2]
```diff
@@ -113,6 +124,7 @@ $events = db_QueryFetch(
 	);
 
 $eventindex = count($events) + 1;
+$eventsId = $events_root[0]['id'];
```
Comment (mikekasprzak) [inline sandbox/simulator/main]: Modified date is unused, you use a LIMIT 1 in the query, and you only care about a single result value. In this case, you can use `db_QueryFetchValue`, and store the returned value in `$eventsId` directly.

File: sandbox/simulator/main (hunk 1 above)
Comment (local-minimum) [reply sandbox/simulator/main]: fixed.

File: sandbox/simulator/main (hunk 2 above)
Comment (local-minimum) [reply sandbox/simulator/main]: fixed

**Pull request merged**